### ISSUE TITLE
Error sourced init orgs --token

### ISSUE THREAD

**Issue by clbaeza** [src-d/sourced-ce#225]
Error when I try to use init. I get the same with local repo.
I downloaded sourced-ce_v0.15.0_darwin_amd64.tar on macOS Mojave 10.14.6 with docker desktop 2.1.0.1
![error](https://user-images.githubusercontent.com/4458573/63547908-26e6b500-c4fb-11e9-8ca1-1fbe39526962.png)

**Comment by smacker** [comment]
Hi @clbaeza !
Could you please share the output of this commands?

Docker-compose version:
```
docker-compose -v
```

`.env` file:
```
cat ~/.sourced/workdirs/__active__/.env
```

**Comment by clbaeza** [comment]
Hi, here are the information you required. thanks!

docker-compose -v

> docker-compose version 1.24.1, build 4667896b

cat ~/.sourced/workdirs/__active__/.env

> COMPOSE_PROJECT_NAME=srcd-b3BlbmRpY29t
> GITBASE_VOLUME_TYPE=volume
> GITBASE_VOLUME_SOURCE=gitbase_repositories
> GITBASE_SIVA=true
> GITHUB_ORGANIZATIONS=opendicom
> GITHUB_TOKEN=<token>
> NO_FORKS=true
> GITBASE_LIMIT_CPU=
> GITCOLLECTOR_LIMIT_CPU=-0.1
> GITBASE_LIMIT_MEM=

**Comment by smacker** [comment]
@clbaeza thank you. #226 should fix the problem. But could you please show output of this command:
```
docker info --format "{{.NCPU}}"
```

While there is no new release with the fix, as a temporary workaround you can edit `~/.sourced/workdirs/__active__/.env` by changing
```
-GITCOLLECTOR_LIMIT_CPU=-0.1
+GITCOLLECTOR_LIMIT_CPU=
```

And then
```
cd ~/.sourced/workdirs/__active__
docker-compose --compatibility up
```

UI will be available after 30-60 seconds at 
```
  http://127.0.0.1:8088
  user:admin
  pass:admin
```

P.S. I changed your comment to remove token.

**Comment by smacker** [comment]
That is interesting. The only incorrect value in your config is `GITCOLLECTOR_LIMIT_CPU`.

Could you validate that `docker-compose` works on your machine? Because the error doesn't contain any information about incorrect configuration except some internal traceback.

```
mkdir lets_test
cd  lets_test
touch docker-compose.yml
```

put 
```yml
version: '3'
services:
  redis:
    image: "redis:alpine"
```

into newly created `docker-compose.yml` and run `docker-compose up` from the same directory. Does it work?

**Comment by clbaeza** [comment]
Works without a problems.

![test_redis](https://user-images.githubusercontent.com/4458573/63619334-55789480-c5bc-11e9-8deb-3648b41303bf.png)

**Comment by smacker** [comment]
Thank you for helping with debugging.

Please execute:
```
sourced compose download https://gist.githubusercontent.com/smacker/6e4dc3fe5c666d4ad8604334c2664ecb/raw/0bf245ec499cacf8262f7500981020897960aa7c/docker-compose-no-limits.yml
```

After that sourced should work for you normally. (no need to change `.env` file or run docker-compose manually)

**Comment by clbaeza** [comment]
Now it's works, thanks for all!!!

**Comment by carlosms** [comment]
This should be now properly fixed in the [v0.15.1 release](https://github.com/src-d/sourced-ce/releases/tag/v0.15.1).
Thanks for reporting this @clbaeza.
